src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,41 @@
+package codeu.controller;
+
+import java.io.IOException;
```

Comment:
As @vocatwk mentioned, I'd go over these to make sure nothing is being imported unnecessarily.

Reply:
Will do, thank you!